packages/react-reconciler/src/ReactFiberFlags.js

```diff
@@ -10,50 +10,53 @@
-export const PassiveUnmountPendingDev = /*     */ 0b0010000000000000;
-export const Hydrating = /*                    */ 0b0000010000000000;
-export const HydratingAndUpdate = /*           */ 0b0000010000000100;
+export const PassiveUnmountPendingDev = /*     */ 0b000010000000000000;
+export const Hydrating = /*                    */ 0b000000010000000000;
+export const HydratingAndUpdate = /*           */ 0b000000010000000100;
 
 // Passive & Update & Callback & Ref & Snapshot
-export const LifecycleEffectMask = /*          */ 0b0000001110100100;
+export const LifecycleEffectMask = /*          */ 0b000000001110100100;
 
 // Union of all host effects
-export const HostEffectMask = /*               */ 0b0000011111111111;
+export const HostEffectMask = /*               */ 0b000000011111111111;
 
 // These are not really side effects, but we still reuse this field.
-export const Incomplete = /*                   */ 0b0000100000000000;
-export const ShouldCapture = /*                */ 0b0001000000000000;
-export const ForceUpdateForLegacySuspense = /* */ 0b0100000000000000;
+export const Incomplete = /*                   */ 0b000000100000000000;
+export const ShouldCapture = /*                */ 0b000001000000000000;
+export const ForceUpdateForLegacySuspense = /* */ 0b000100000000000000;
 
 // Static tags describe aspects of a fiber that are not specific to a render,
 // e.g. a fiber uses a passive effect (even if there are no updates on this particular render).
 // This enables us to defer more work in the unmount case,
 // since we can defer traversing the tree during layout to look for Passive effects,
 // and instead rely on the static flag as a signal that there may be cleanup work.
-export const PassiveStatic = /*                */ 0b1000000000000000;
+export const PassiveStatic = /*                */ 0b001000000000000000;
 
 // Union of side effect groupings as pertains to subtreeFlags
-export const BeforeMutationMask = /*           */ 0b0000001100001010;
-export const MutationMask = /*                 */ 0b0000010010011110;
-export const LayoutMask = /*                   */ 0b0000000010100100;
-export const PassiveMask = /*                  */ 0b0000001000001000;
+export const BeforeMutationMask = /*           */ 0b000000001100001010;
+export const MutationMask = /*                 */ 0b000000010010011110;
+export const LayoutMask = /*                   */ 0b000000000010100100;
+export const PassiveMask = /*                  */ 0b000000001000001000;
 
 // Union of tags that don't get reset on clones.
 // This allows certain concepts to persist without recalculting them,
 // e.g. whether a subtree contains passive effects or portals.
-export const StaticMask = /*                   */ 0b1000000000000000;
+export const StaticMask = /*                   */ 0b001000000000000000;
+
+export const MountLayoutDev = /*               */ 0b010000000000000000;
+export const MountPassiveDev = /*              */ 0b100000000000000000;
```

Comment:
Initially, I found myself wondering why these two flags were necessary. I think what they enable us to do is to avoid traversing the entire tree after each commit just to see if any of the nested components were newly mounted. (Checking the `Update` or `Passive` effects alone wouldn't tell us enough info.)

Maybe we can add a small explanatory comment above the flags in case anyone else wonders about this?